.github/workflows/ros.yml

```diff
@@ -128,7 +131,10 @@ jobs:
         run: |
           source /opt/ros/$ROS_DISTRO/setup.bash
           cd /kpsr-ros/src/build
+          make tests
+          roscore &
           make test
+          killall roscore
```

Comment:
@tomasz3dk This line fails and is obviously wrong. What would be the correct way to kill the roscore process from line 135?

Reply:
Just remove this line, and run roscore in background. After `make test` will finish with some exit code the runner after that should move to the next step, or stop the container if there is no next steps.